trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -33,8 +33,8 @@ const updateReportStatusPayloadValidation = [
   checkSchema({
     'reportsWithStatus.*.status': {
       isIn: {
-        options: `${UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED},${UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_COMPLETED}`,
-        errorMessage: `Report status must be one of the following: ${UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED},${UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_COMPLETED}`,
+        options: `${UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED},${UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_COMPLETED},${UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION}`,
```

Comment:
I originally didn't put it into a seperate file because the rest of the validation chains exist only in this file, but I agree that it should really be in its own - will make the change